stable/mlrun/templates/db-deployment.yaml

```diff
@@ -0,0 +1,107 @@
+    spec:
+    {{- with .Values.db.image.pullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      securityContext:
+        {{- toYaml .Values.db.podSecurityContext | nindent 8 }}
+      initContainers:
+        - name: init-mysql
+          image: "{{ .Values.db.image.repository }}:{{ .Values.db.image.tag }}"
+          imagePullPolicy: {{ .Values.db.image.pullPolicy }}
+          resources:
+            {{- toYaml .Values.db.resources | nindent 12 }}
+          env:
+            - name: MYSQL_ALLOW_EMPTY_PASSWORD
+              value: "true"
+          command: ["/bin/bash", "/etc/config/mysql/init-v3io-mysql.sh"]
+          securityContext:
+            runAsUser: 1001
+          volumeMounts:
+            - name: mysql-fuse
+              mountPath: "/var/lib/mysql"
+            - name: mysql-config
+              mountPath: /etc/config/mysql
+      containers:
+        - name: {{ template "mlrun.name" . }}-{{ .Values.db.name }}
+          securityContext:
+            {{- toYaml .Values.db.securityContext | nindent 12 }}
+          image: "{{ .Values.db.image.repository }}:{{ .Values.db.image.tag }}"
+          imagePullPolicy: {{ .Values.db.image.pullPolicy }}
+          command: ["/bin/bash", "/etc/config/mysql/v3io-mysql.sh"]
+          ports:
+          - name: mysql
+            containerPort: 3306
+            protocol: TCP
+          {{- if .Values.db.livenessProbe }}
+          livenessProbe:
+            {{ toYaml .Values.db.livenessProbe | nindent 12 }}
+          {{- end }}
+          {{- if .Values.db.readinessProbe }}
+          readinessProbe:
+            {{ toYaml .Values.db.readinessProbe | nindent 12 }}
+          {{- end }}
+          env:
+          - name: MYSQL_ALLOW_EMPTY_PASSWORD
+            value: "true"
+          {{- if .Values.db.extraEnv }}
+          {{ toYaml .Values.db.extraEnv | nindent 10 }}
+          {{- end }}
+          {{- if .Values.db.envFrom }}
+          envFrom:
+          {{ toYaml .Values.db.envFrom | nindent 10 }}
+          {{- end }}
+          resources:
+            {{- toYaml .Values.db.resources | nindent 12 }}
+          volumeMounts:
+          - name: mysql-fuse
+            mountPath: "/var/lib/mysql"
+          - name: mysql-config
+            mountPath: /etc/config/mysql
+          - name: mysql-init-scripts
+            mountPath: /etc/config/mysql/init-scripts
+      volumes:
+        - name: mysql-init-scripts
+          configMap:
+            name: {{ include "mlrun.db.initConfigMapName" . }}
+        - name: mysql-config
+          configMap:
+            name: {{ include "mlrun.db.fullname" . }}
+        # Add MySQL data mount (Fuse)
+        - name: mysql-fuse
+          flexVolume:
+            driver: "v3io/fuse"
+            options:
+              container: users
+              subPath: /{{ .Values.v3io.username }}/.mlrun/mysql
+            secretRef:
+              name: {{ .Release.Name }}-v3io-fuse
```

Comment:
We'll also need to support mysql over pvc, I'm ok with it being phase 2, just making sure you're aware